scipy/sparse/linalg/_norm.py

```diff
@@ -137,9 +144,13 @@ def norm(x, ord=None, axis=None):
                              (axis, x.shape))
         if row_axis % nd == col_axis % nd:
             raise ValueError('Duplicate axes given.')
-        if ord == 2:
-            raise NotImplementedError
-            #return _multi_svd_norm(x, row_axis, col_axis, amax)
+        if ord in (2, 'spec'):
+            # spectral norm
+            from scipy.sparse import csc_matrix 
+            from scipy.sparse.linalg import svds
+            x = csc_matrix(x, dtype=float)
```

Comment:
The conversion to `csc_matrix` shouldn't be necessary.
Moreover, one also has to be able to compute norms of complex matrices.